### DISCUSSION TITLE
Is there a way to extract chat data automatically?

### DISCUSSION THREAD

**Discussion by KhalfaniW** [lobehub/lobehub#2833] [General Question] [answered]
I'm looking for a way to export all agents and message automatically without opening the UI.

For example if there was a `/data` path that one could http request to get a json of chat data.

**Comment by KhalfaniW** [comment]
Or is there a way I could pretend to be WebRTC client so I could get a copy of the synchronization data and save it as a json?

**Reply by arvinxx** [reply, marked as answer]
You can use server database

**Reply by KhalfaniW** [reply]
Thank you

**Reply by ez-as2** [reply]
I
